homeassistant/components/incomfort/strings.json

```diff
@@ -20,8 +20,9 @@
       "auth_error": "Invalid credentials.",
       "no_heaters": "No heaters found.",
       "not_found": "No Lan2RF gateway found.",
-      "timeout_error": "Time out when connection to Lan2RF gateway.",
-      "unknown": "Unknown error when connection to Lan2RF gateway."
+      "timeout_error": "Time out when connecting to Lan2RF gateway.",
+      "reconfigure_successful": "[%key:common::config_flow::abort::reconfigure_successful%]",
+      "unknown": "Unknown error when connecting to Lan2RF gateway."
```

Comment:
we never abort with unknown right?

Reply:
You are right. Same applie to some other errors. We should address this in a different PR. As the changes here are just about formatting/sorting.